docs/development/getting_started.rst

```diff
@@ -23,7 +23,7 @@ Ubuntu/Debian
 
    sudo apt-get install -y dpkg-dev virtualbox-dkms linux-headers-$(uname -r) build-essential git
 
-We recommend using the latest stable version of Vagrant (``1.7.4`` at the time
+We recommend using the latest stable version of Vagrant (``1.8.4`` at the time
```

Comment:
Tested, updated docs to 1.8.5. Fell back to 1.8.4 on my workstation so I can make use of apt updates going forward.